### PR TITLE
feat(escuelafrontend): dynamic ogImages 

### PR DESCRIPTION
**Pull request by laurosilvacom**
Frame 1

gif

### CODE REVIEW

File: apps/escuelafrontend/src/pages/api/opengraph.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      ? `#60dafa`
+      : tagSlug == 'javascript'
+      ? `#f0db4e`
+      : tagSlug == 'typescript'
+      ? `#3077c6`
+      : tagSlug == 'npm'
+      ? `#d60100`
+      : tagSlug == 'next-js'
+      ? `#565656`
+      : `#565656`
```
Comment (laurosilvacom) [inline apps/escuelafrontend/src/pages/api/opengraph.ts:35]: @NicollGuarnizo this updates the dynamic colors on the cards

File: apps/escuelafrontend/src/pages/api/opengraph.ts [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+  await page.setContent(
+    contentType == 'article'
+      ? article
+      : contentType == 'profile'
+      ? profile
+      : contentType == 'course'
+      ? course
+      : course,
+  )
```
Comment (laurosilvacom) [inline apps/escuelafrontend/src/pages/api/opengraph.ts:209]: depending on the content type we render different layout

File: apps/escuelafrontend/src/pages/api/opengraph.ts [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+              <div>
+                 <img src="${instructorImage}" class="h-20 w-20 rounded-full" />
+              </div>
+              <p class="text-white font-semibold text-2xl leading-tighter font-sans ml-2">${instructorName}</p>
+           </div>
+        </div>
+        <div class="banner"></div>
+     </body>
+  </html>
+`.trim()
```
Comment (laurosilvacom) [inline apps/escuelafrontend/src/pages/api/opengraph.ts:89]: this is HTML that creates the structure for the cards

File: apps/escuelafrontend/src/layouts/article.tsx [hunk 4]
```diff
@@ -38,12 +60,16 @@ const ArticleTemplate: React.FC<ArticleTemplateProps> = ({meta}) => {
...
-              alt: title,
-            },
-          ],
+          type: 'article',
+          article: {
+            publishedTime: publishedTime,
+            modifiedTime: modifiedTime,
+            authors: [instructor.name],
+            tags: [tag.name],
+          },
```
Comment (laurosilvacom) [inline apps/escuelafrontend/src/layouts/article.tsx:69]: updates articles to include `publishedTime` and `modifiedTime` for better SEO

**Review by NicollGuarnizo — APPROVED** [review]

**Pull request merged**